tests/unit/core/reducers/test_addons.js

```diff
@@ -277,28 +296,22 @@ describe(__filename, () => {
         undefined,
         fetchAddon({ slug, errorHandler: createStubErrorHandler() }),
       );
-      expect(state.loadingBySlug[slug]).toBe(true);
-    });
-
-    it('is case insensitive', () => {
-      const slug = 'some-slug';
-      const state = addons(
-        undefined,
-        fetchAddon({
-          slug: slug.toUpperCase(),
-          errorHandler: createStubErrorHandler(),
-        }),
-      );
-      expect(state.loadingBySlug).toHaveProperty(slug);
+      expect(state.loadingByIdInURL[slug]).toBe(true);
     });
   });
 
   describe('loadAddon', () => {
-    it('requires addons', () => {
+    it('requires an addon', () => {
       expect(() => {
         loadAddon();
       }).toThrow('addon is required');
     });
+
+    it('requires a slug', () => {
```

Comment:
We generally don't add tests for `invariant`s. I know there was one already for "requires an addon", but maybe we'd be better removing that one, than adding another?

Reply:
yep I am fine with that